invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/oaipmh/details/LinksTable.js

```diff
@@ -80,11 +80,17 @@ class LinksTable extends Component {
    * Used to update links when the metadata prefix is changed.
    */
   replaceLinkPrefix = (link, newPrefix) => {
-    const oldPrefix = this.getPrefixFromLink(link);
-    if (oldPrefix) {
-      return link.replace(oldPrefix, newPrefix);
+    if (_isEmpty(link)) return null;
+
+    const prefixParam = "metadataPrefix";
+    const url = new URL(link);
+    const params = url.searchParams;
+
+    if (params.has(prefixParam)) {
```

Comment:
I am not sure if we need to check if the prefix is there, just replicating old behaviour 🤔